tests/unit/src/test/scala/tests/StatusBarLspSuite.scala

```diff
@@ -16,7 +16,9 @@ class StatusBarLspSuite extends BaseLspSuite("status-bar") {
           |}
         """.stripMargin
       )
-      _ <- server.didOpen("a/src/main/scala/A.scala")
+      _ <- server.didSave("a/src/main/scala/A.scala") { text =>
```

Comment:
Looks like this was actually compiled at `cleanCompileCache("a")` before and since we no longer send notification on noop compilation the test started failing.